plugins/nv-filters/nvvfx-load.h

```diff
@@ -38,8 +38,10 @@ extern "C" {
 
 #define CUDARTAPI
 
-#ifdef LIBNVVFX_ENABLED
 #define MIN_VFX_SDK_VERSION (0 << 24 | 7 << 16 | 2 << 8 | 0 << 0)
+#define MIN_AFX_SDK_VERSION (1 << 24 | 3 << 16 | 0 << 0)
```

Comment:
See other question about having to update this in two places.

Reply:
done